### ISSUE TITLE
Release stable versions of userfrosting modules

### ISSUE THREAD

**Issue by alexweissman** [userfrosting/UserFrosting#622]
Right now, we're using dev versions of the following repos:

```
"userfrosting/assets": "1.0.x-dev",
"userfrosting/config": "dev-master",
"userfrosting/fortress": "dev-master",
"userfrosting/i18n": "1.0.x-dev",
"userfrosting/session": "1.0.x-dev"
```

We should figure out if we want to make any more major changes to them, and then release stable versions.

**Comment by lcharette** [comment]
`i18n` is final. We can make a `x.0.1` release for it if a bug is found.

**Comment by alexweissman** [comment]
What if we need to modify `i18n` for #621 ?

**Comment by lcharette** [comment]
I've merged and created tags for every UF components and updated the main repo `composer.json` file.

As discussed with @alexweissman, we fixed every component starting version at `4.0.0` and they should be increased as necessary when *they* are updated. This means they won't get bumped with every UF minor release.

If someone encounter an error when updating composer, they should checkout the latest UF version and run `composer clear-cache` to fix everything.
